lib/media/track/mediatrack.js

```diff
@@ -145,27 +151,35 @@ class MediaTrack extends Track {
   _end() {
     this._log.debug('Ended');
     if (this._dummyEl) {
-      this._detachElement(this._dummyEl);
+      this._detachElementInternal(this._dummyEl);
+      this._dummyEl.remove();
+      this._dummyEl.srcObject = null;
       this._dummyEl.oncanplay = null;
+      this._dummyEl = null;
     }
   }
 
   attach(el) {
-    if (typeof el === 'string') {
-      el = this._selectElement(el);
-    } else if (!el) {
-      el = this._createElement();
-    }
-    this._log.debug('Attempting to attach to element:', el);
-    el = this._attach(el);
-
-    if (this._shouldShimAttachedElements && !this._elShims.has(el)) {
-      const onUnintentionallyPaused = this._playPausedElementsIfNotBackgrounded
-        ? () => playIfPausedAndNotBackgrounded(el, this._log)
-        : null;
-      this._elShims.set(el, shimMediaElement(el, onUnintentionallyPaused));
+    try {
```

Comment:
Is this try catch block for debugging? Should we remove this?

Reply:
I would keep this block, since the issue will still happen when 75+ elements are created.

Reply:
Should we throw a TwilioError instead? That way, it follows our convention when an error happens.

Reply:
TwilioErrors are heavy - require associated documentation and such. I think this error is fine for now. I am also hoping that this issue will get fixed by chrome as lot of apps are affected by it. https://bugs.chromium.org/p/chromium/issues/detail?id=1232649

Reply:
Update: I found that the try block is not very useful as the error happens async and not caught by the block :(. I will remove the try block.